README.md

```diff
@@ -25,10 +22,11 @@ This app is built using Ruby and Sinatra.  It makes use of GitHub webhooks to re
 #### Setting up the app
 1. Clone the repo down to your local machine.
 2. Run `bundle install`.
-3. Get your [GitHub access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/) and assign to the `GITHUB_ACCESS_TOKEN` environment variable.
-4. Get your [Trello developer API key](https://trello.com/app-key) and assign to the `TRELLO_PUBLIC_KEY` environment variable. You may also find this key using [ruby-trello's](https://github.com/jeremytregunna/ruby-trello) helpful methods.
-5. Get your [Trello user token](https://developers.trello.com/authorize) and assign to the `TRELLO_MEMBER_TOKEN` environment variable. You may also find this token using [ruby-trello's](https://github.com/jeremytregunna/ruby-trello) helpful methods.
-6. To set up a Webhook:
+3. Get your [GitHub access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/) and assign to the `GITHUB_ACCESS_TOKEN` environment variable. On GOV.UK, we use a token associated with the govuk-ci user.
+4. Create a Trello account and add it to your team's Trello board, giving read and write access. On GOV.UK, we use the [@pullrequestposter](https://trello.com/u/pullrequestposter) user.
+5. Get your [Trello developer API key](https://trello.com/app-key) and assign to the `TRELLO_PUBLIC_KEY` environment variable. You may also find this key using [ruby-trello's](https://github.com/jeremytregunna/ruby-trello) helpful methods.
+6. Get your [Trello user token](https://developers.trello.com/authorize) and assign to the `TRELLO_MEMBER_TOKEN` environment variable. You may also find this token using [ruby-trello's](https://github.com/jeremytregunna/ruby-trello) helpful methods.
+7. To set up a Webhook:
```

Comment:
Perhaps for more practical instructions on setting up for a new gov.uk app we could include the Webhook example (https://github.com/alphagov/publishing-api/settings/hooks), which would also answer the "Where do I get payload URL from?" question?

Reply:
Ah actually I was going to document that it gets set up automatically through govuk-saas-config, but forgot to do that! Will do it now.